xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecStarfish.cpp

```diff
@@ -353,7 +364,11 @@ bool CDVDVideoCodecStarfish::AddData(const DemuxPacket& packet)
     std::string result = m_starfishMediaAPI->Feed(json.c_str());
 
     if (result.find("Ok") != std::string::npos)
+    {
+      if (startPlaying)
+        m_starfishResource->StartPlaying();
```

Comment:
Is this here placed correctly? I could imagine that it'd be better placed when a `PF_EVENT_TYPE_STR_STATE_UPDATE__LOADCOMPLETED` or `PF_EVENT_TYPE_STR_STATE_UPDATE__PLAYING` is received. Maybe also call `AcbAPI_setState(acbId, APPSTATE_FOREGROUND, PLAYSTATE_PAUSED, NULL);` when `PF_EVENT_TYPE_STR_STATE_UPDATE__PAUSED` is received.